content/direct-deposit-management/release-notes/2024-11-27.md

```diff
@@ -0,0 +1,3 @@
+We added an alert notification for direct deposit accounts flagged for “potential” fraud. When a backend VA data source has an account flagged for potential fraud activity, the API response includes an HTTP XXX alert message.
+
+Access to deposit accounts flagged for fraud is not blocked by the API; rather, the HTTP alert messaging notifies when access attempts occur for flagged accounts. This puts control for blocking account access or monitoring activity with the API requestor.
```

Comment:
Is "API requestor" the same thing as a consumer?

Reply:
Yes it's the consumer.